static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.spec.js

```diff
@@ -111,6 +119,16 @@ describe('First PV consents banner', () => {
             });
             return expect(await banner.canShow()).toBe(false);
         });
+        it('should render outside the EU if in test', async () => {
+            isInVariantSynchronous.mockImplementation(
+                (testId, variantId) => variantId === 'variant'
+            );
+            getCookie.mockImplementation(_ => {
+                if (_ === 'GU_geo_continent') return '??';
+                return null;
+            });
+            return expect(await banner.canShow()).toBe(true);
+        });
```

Comment:
Minor: maybe overkill but this would be a chance to cover other possible cases

```js
        it('should not render outside the EU when commercial consent test participation is "control"', async () => {
            isInVariantSynchronous.mockImplementation(
                (testId, variantId) => variantId === 'control'
            );
            getCookie.mockImplementation(_ => {
                if (_ === 'GU_geo_continent') return '??';
                return null;
            });
            return expect(await banner.canShow()).toBe(false);
        });
```

N.B. I have not actually ran this test 😅